torchaudio/pipelines/_wav2vec2/impl.py

```diff
@@ -347,6 +372,7 @@ def _get_state_dict(self, dl_kwargs):
     "wav2vec2_fairseq_large_ls960.pth",
     {
         "extractor_mode": "group_norm",
+        "normalize_waveform": True,
```

Comment:
Could you move this parameter outside of `_params` and define a separate property directly on `Wav2Vec2Bundle`?

Reply:
I just realized that the current suggestion is following what was done for "model_type" parameter. Sorry that one should not be done like that. It should be outside of `_params` attribute @nateanl https://github.com/pytorch/audio/pull/2833/files#r1034111931

Reply:
sure, will make a fix for `model_type` in a separate PR.